docs/software/inkhud/index.mdx

```diff
@@ -0,0 +1,148 @@
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+## What is InkHUD?
+
+InkHUD is a user interface for Meshtastic devices which have an E-Ink display. It is designed to give a quick "heads-up", showing important information at a glance, complimenting a connected [app](/downloads/). Specifically targeting E-Ink displays, it is offered as an alternative to the original OLED user interface.
+
+Under active development since late 2024, the list of features and supported devices will continue to grow.
+
+## Supported Devices
+As of March 2025. Watch this space 👀
+
+- **Complete Devices**
+  - LilyGo T-Echo
+  - Heltec Wireless Paper
+  - Heltec Vision Master E213
+  - Heltec Vision Master E290
+- **DIY Displays** (custom builds, _for the brave.._)
+  - WeAct Studio 4.2" Black & White
+
+## Installation
+
+InkHUD can be installed on supported devices using the [Meshtastic Web Flasher](https://flasher.meshtastic.org). Select a 2.6+ version firmware, and toggle the `Install InkHUD display` option on the flash screen.
+
+Prebuilt firmware binaries are also available on the [downloads page](/downloads/). The InkHUD versions are identified by the **"-inkhud"** suffix in the filename.
+
+If you wish to use InkHUD with a DIY Meshtastic device, you will need to [build the firmware manually](/docs/development/firmware/build/), specially configuring InkHUD for your hardware.
+{/* Todo: link to InkHUD's maintainer doc (readme in firmware repo), once merged. */}
+
+## Initial Setup
+
+* [Set your LoRa region](/docs/configuration/region-by-country/)
+* [Set your timezone](/docs/configuration/radio/device#tzdef-timezone-definition)
+
+## Buttons
+
+InkHUD is designed to be used with just one button. A mixture of short and long button presses are use to control the user interface.
+
+## Applets
+
+InkHUD provides a set of "applets", each one showing a different set of information. You can choose which applets to enable using the on-screen menu. For in-depth descriptions, see the [List of InkHUD Applets](./applets.mdx).
+
+**Short-press** the user button to switch to the next applet.
+
+<div align="center">
+  <img
+    src="/img/software/inkhud/applet-cycle.gif"
+    alt="animation of cycling though inkhud applets with user button"
+    style={{maxHeight: '256px', width:'auto' }}
+  />
+</div>
+
+## On-Screen Menu
+
+InkHUD has a built in menu, with a few basic actions, and options for configuring the device.
+
+To open the on-screen menu, **long-press** the user button.
+* **short-press**: next item
+* **long-press**: select
+
+<div align="center">
+  <img
+    src="/img/software/inkhud/menu.gif"
+    alt="animation of navigating inkhud menu"
+    style={{maxHeight: '256px', marginBottom: '20px'}}
+  />
+</div>
+
+:::tip
+Only wanted a quick peek at the clock? Long-press a second time to close the menu. 
+:::
+
+### Options
+
+#### Applet Selection
```

Comment:
@pdxlocations @rcarteraz want your thoughts here

Not sure if using h4's is a pattern we're following already and this just sticks to that, but having each one of these as an h4 seems not quite right. I think that using a table or just bullet points could be better.

What are y'alls thoughts?